docs/pages/web/tableofcontents.js

```diff
@@ -0,0 +1,186 @@
+                layout="column"
+                hideEditor
+              />
+            }
+          />
+          <MainSection.Card
+            cardSize="md"
+            type="don't"
+            description="Place the TableOfContents really far from the main content making it easy to miss, or hard to move efficiently between it and the content."
+            sandpackExample={
+              <SandpackExample
+                code={dontPlaceFarAway}
+                name="Don't - Place the TableOfContents really far from the main content"
+                layout="column"
+                hideEditor
+                hideControls
+              />
+            }
+          />
+        </MainSection.Subsection>
+        <MainSection.Subsection columns={2}>
+          <MainSection.Card
+            cardSize="md"
+            type="do"
+            description="Place the TableOfContents to the right of the main content on a LTR surface and to the left of the main content on an RTL surface."
+          />
+          <MainSection.Card
+            cardSize="md"
+            type="don't"
+            description="Place the TableOfContents right next to the SideNavigation to differentiate it from link navigation."
+          />
+          <MainSection.Card
+            cardSize="md"
+            type="do"
+            description="Use one TableOfContents per page."
+          />
+          <MainSection.Card
+            cardSize="md"
+            type="don't"
+            description="Introduce unnecessary complexity by using more than one TableOfContents on a page."
+          />
+        </MainSection.Subsection>
+      </MainSection>
+
+      <MainSection name="Subcomponents">
+        <MainSection.Subsection
+          title={generatedDocGen.TableOfContentsItem?.displayName}
+          description={generatedDocGen.TableOfContentsItem?.description}
+        >
+          <GeneratedPropTable
+            name={generatedDocGen.TableOfContentsItem?.displayName}
+            id={generatedDocGen.TableOfContentsItem?.displayName}
+            generatedDocGen={generatedDocGen.TableOfContentsItem}
+          />
+        </MainSection.Subsection>
+      </MainSection>
+
+      <AccessibilitySection
+        name={generatedDocGen?.TableOfContents.displayName}
+        description="Be sure to include an `accessibilityLabel` for the screen reader for each item in the TableOfContents"
+      />
+
+      <MainSection
+        name="Localization"
+        description="Be sure to localize text and `accessibilityLabel`. Note that localization can lengthen text by 20 to 30 percent."
+      />
+
+      <MainSection name="Variants">
+        <MainSection.Subsection
+          title="Nested directory"
+          description="TableOfContents supports 5 levels of nesting. The first level maps to a section’s heading, which is usually an H3. The second level maps to a section’s subheading, which is usually an H4."
```

Comment:
The first level maps to a section’s heading, which is usually an H3.

H3 or H2?

I would think it should be an H2 no? The top section is a H1, for example  PageHeader. 

Subsequent heading just be H2... why H3?

Reply:
I copied that from the coda doc. Didn't realy bother with this much. But what you said makes sense. I will update it